config/server.lua

```diff
@@ -25,7 +25,7 @@ return {
         identifierTypes = {
             citizenid = {
                 valueFunction = function()
-                    return tostring(RandomLetter(3) .. RandomNumber(5)):upper()
+                    return tostring(lib.string.random('A', 3) .. lib.string.random(1, 2))
```

Comment:
```suggestion
                    return tostring(lib.string.random('A', 3) .. lib.string.random('1', 2))
```
ox docs say that the first param is a string. Also, isn't this a change in behavior? The previous value function was five random numbers and now this is two?